Gems/EMotionFX/Code/Source/Integration/Components/SimpleLODComponent.cpp

```diff
@@ -142,6 +143,12 @@ namespace EMotionFX
         {
             ActorComponentNotificationBus::Handler::BusConnect(GetEntityId());
             AZ::TickBus::Handler::BusConnect();
+
+            // Disable the automatic mesh LOD level adjustment based on screen space in case a simple LOD component is present.
+            // The simple LOD component overrides the mesh LOD level and syncs the skeleton with the mesh LOD level.
+            AZ::Render::MeshComponentRequestBus::Event(GetEntityId(),
+                &AZ::Render::MeshComponentRequestBus::Events::SetLodType,
+                AZ::RPI::Cullable::LodType::SpecificLod);
```

Comment:
Added that. I was thinking about that but didn't add it as it would only capture the time before the component activation which happens rarely after the actor component activation.